bundle-size/app.js

```diff
@@ -130,28 +130,88 @@ function formatBundleSizeDelta(delta) {
+      `size of \`v0.js\`) by ${baseRuntimeDeltaFormatted}. As part of an ` +
+      'ongoing effort to reduce the bundle size, this change requires special' +
+      'approval.\n' +
+      'A member of the bundle-size group will be added automatically to ' +
+      'review this PR. Only once the member approves this PR, can it be ' +
+      'merged. If you do not receive a response from the group member, feel ' +
+      `free to tag another person in ${process.env.REVIEWER_TEAM_NAMES}` +
+      appendExtraBundleSizes(otherBundleSizeDeltas, missingBundleSizes),
+  };
+}
+
 /**
  * Returns an encouraging result summary when the bundle size is reduced.
  * Otherwise the summary is neutral, but not discouraging.
  *
- * @param {number} delta bundle size delta.
- * @param {string} deltaFormatted bundle size delta from `formatBundleSizeDelta`
- * @return {string} successful summary message.
+ * @param {number} baseRuntimeDelta bundle size delta.
+ * @param {Array<string>} otherBundleSizeDeltas text description of other bundle
+ *   size changes.
+ * @param {Array<string>} missingBundleSizes text description of missing bundle
+ *   sizes from other `master` or the pull request.
+ * @return {{title: string, summary: string}} check output.
  */
-function successfulSummaryMessage(delta, deltaFormatted) {
-  if (delta <= HUMAN_ENCOURAGEMENT_MAX_DELTA) {
-    return (
-      'This pull request *reduces* the bundle size ' +
-      '(brotli compressed size of `v0.js`), so no special approval is ' +
-      `necessary. The bundle size change is ${deltaFormatted}.`
-    );
+function successfulCheckOutput(
+  baseRuntimeDelta,
+  otherBundleSizeDeltas,
+  missingBundleSizes
+) {
+  const baseRuntimeDeltaFormatted = formatBundleSizeDelta(baseRuntimeDelta);
+  const title = `${baseRuntimeDeltaFormatted} | no approval necessary`;
+  let summary;
+  if (baseRuntimeDelta <= HUMAN_ENCOURAGEMENT_MAX_DELTA) {
+    summary =
+      'This pull request *reduces* the bundle size (brotli compressed size ' +
+      'of `v0.js`), so no special approval is necessary. The bundle size ' +
+      `change is ${baseRuntimeDeltaFormatted}.`;
+  } else {
+    summary =
+      'This pull request does not change the bundle size (brotli compressed ' +
+      'size of `v0.js`) by any significant amount, so no special approval is ' +
+      `necessary. The bundle size change is ${baseRuntimeDeltaFormatted}.`;
   }
 
+  summary += appendExtraBundleSizes(otherBundleSizeDeltas, missingBundleSizes);
+
+  return {title, summary};
+}
+
+/**
+ * Return formatted extra changes to append to the check output summary.
+ *
+ * @param {Array<string>} otherBundleSizeDeltas text description of other bundle
+ *   size changes.
+ * @param {Array<string>} missingBundleSizes text description of missing bundle
+ *   sizes from other `master` or the pull request.
+ * @return {string} formatted extra changes;
+ */
+function appendExtraBundleSizes(otherBundleSizeDeltas, missingBundleSizes) {
```

Comment:
Since this function is `append` the verb, I expected it to take a summary and return something appended to it. I'd either rename to just `extraBundleSizes` or do like I mentioned and return `summary: appendExtraBundleSizes(summary, ...)`

Reply:
Renamed to `extraBundleSizesSummary`